pkg/virt-controller/services/template.go

```diff
@@ -62,6 +70,66 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VM) (*kubev1.Pod, error) {
 	return &pod, nil
 }
 
+func (t *templateService) RenderMigrationJob(vm *v1.VM, sourceNode *kubev1.Node, targetNode *kubev1.Node) (*batch.Job, error) {
+	found := false
+	srcAddr := ""
+	dstAddr := ""
+	for _, addr := range sourceNode.Status.Addresses {
+		if addr.Type == kubev1.NodeExternalIP {
+			srcAddr = addr.Address
+			found = true
+		}
+	}
+	if !found {
```

Comment:
you can just check with if srcAddr == "". That's also more correct since addr.Address could be an empty string too ...